modin/pandas/series.py

```diff
@@ -308,22 +370,39 @@ def __rtruediv__(self, left):
     __itruediv__ = __truediv__
 
     @property
-    def values(self):
+    def values(self):  # noqa: RT01, D200
```

Comment:
pls make docstrings that pass validation without removing checks.

here and below.

Reply:
Docstrings doesn't pass checks because they are constructed in the way described in the [comment](https://github.com/modin-project/modin/pull/3056#discussion_r635174944) (`D200` - One-line docstring should fit on one line with quotes (resulting docstring will not be one-line), `PR01` and `RT01` - absence of `Parameters` and `Returns` description).